storage/replica_command.go

```diff
@@ -546,12 +550,29 @@ func (r *Replica) RangeLookup(batch engine.Engine, args proto.RangeLookupRequest
 		// less than "f", and so the meta row {"f"->["c", "f")} would be
 		// ignored by MVCCReverseScan. In the second case, the range descriptor
 		// received by MVCCScan will be filtered before results are returned.
+		// Example: ranges ["c", "f") and ["f", "z"), reverse scan on ["d", "f"]
```

Comment:
unrelated to this PR, but s/get first the first range/get the first range/ at line 543.

also s/reverse scan on ["d", "f"]/reverse scan on ["d", "f")/ ?